logging/cluster-logging-release-notes.adoc

```diff
@@ -46,42 +46,52 @@ To see these metrics, open the *Administrator* perspective in the {product-title
+
+* Before this update, the Kibana OpenDistro security plugin caused user index migrations to fail. This update resolves the issue by providing a newer version of the plugin. Now, index migrations proceed without errors. (link:https://issues.redhat.com/browse/LOG-1558[LOG-1558])
+
+* Before this update, using a namespace input filter prevented logs in that namespace from appearing in other inputs. With this update, logs are sent to all inputs that can accept them. (link:https://issues.redhat.com/browse/LOG-1570[LOG-1570])
+
+* Before this update, a missing license file for the viaq/logerr dependency caused license scanners to abort without success. With this update, the viaq/logerr dependency is now licensed under Apache 2.0, resolving the issue. (link:https://issues.redhat.com/browse/LOG-1590[LOG-1590])
+
+* Before this update, an incorrect brew tag for curator5 within the  elasticsearch-operator-bundle build pipeline caused the pull of an image pinned to a dummy SHA1. With this update, the build pipeline uses the logging-curator5-rhel8 reference for curator5, and allows index management cronjobs to pull the right image from registry.redhat.io. (link:https://issues.redhat.com/browse/LOG-1624[LOG-1624])
+
+* Before this update, an issue with the `ServiceAccount` permissions caused errors like `no permissions for [indices:admin/aliases/get]`. With this update, a permission fix resolves the issue. (link:https://issues.redhat.com/browse/LOG-1657[LOG-1657])
+
+* Before this update, the Custom Resource Definition (CRD) for the Red Hat OpenShift Logging Operator was missing the Loki output type, which caused the admission controller to reject the `ClusterLogForwarder` custom resource object. With this update, the CRD includes Loki as an output type, so administrators can configure `ClusterLogForwarder` to send logs to a Loki server. (link:https://issues.redhat.com/browse/LOG-1683[LOG-1683])
+
+* Before this update, OpenShift Elasticsearch Operator reconciliation of the `ServiceAccounts` overwrote third-party-owned fields that contained secrets. This issue caused memory and CPU spikes due to frequent secret re-creation. This update resolves the issue: the OpenShift Elasticsearch Operator does not overwrite third-party-owned fields. (link:https://issues.redhat.com/browse/LOG-1714[LOG-1714])
+
+* Before this update, if the `ClusterLogging` custom resource (CR) definition specified a `flush_interval` but did not set the value of `flush_mode` to `interval`, the Fluentd configuration generated an error at runtime. With this update, the Red Hat OpenShift Logging Operator validates the `ClusterLogging` CR and does not generate the Fluentd config if this field mismatch is present. (link:https://issues.redhat.com/browse/LOG-1723[LOG-1723])
```

Comment:
as far as I understand after the update, the CR does not generate an error. If it's true that it doesn't generate the Fluent config I wouldn't abbreviate config here and also specify which configuration.

Reply:
Updated. Thanks.